remoulade/state/middleware.py

```diff
@@ -20,33 +21,41 @@ def __init__(self, backend, state_ttl=3600, max_size=2e6):
         self.state_ttl = state_ttl
         self.max_size = max_size
 
-    def save(self, message, state):
-        args = message.args
-        kwargs = message.kwargs
+    def save(self, message, nameState, **times):
```

Comment:
by convention, we use kwargs for kwargs parameters (rename time to kwargs).
Also use snake case: `state_name`